invenio_preservation_sync/utils.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2024 CERN.
+#
+# Invenio-Preservation-Sync is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see LICENSE file for more
+# details.
+
+"""Utils for Preservation Sync module."""
+
+from flask import current_app, g, url_for
+
+from invenio_preservation_sync.models import PreservationStatus
+
+from .proxies import current_preservation_sync_service as service
+
+
+def preservation_info_render(record):
+    """Render the preservation info."""
+    permissions = record.has_permissions_to(["manage"])
```

Comment:
I think that `manage` won't be enough: this means that record's owners will be able to see the status and click on the link.
However, they will not have access to the Digital Memory platform.

What about using admin permission instead, and not record based?